social/email/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.1.29",
```

Comment:
As per discussions we're going to need to major bump this.

Reply:
I don't usually bump versions in PRs; I let the maintainer decide.  But if you want me to bump it, I can.

Reply:
true - but a) as it's you and b) in this case I want to make sure we get no possible crossover with the 0.1 code.